thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/DetectionJobResource.java

```diff
@@ -231,14 +299,14 @@ public Response generateAnomaliesInRange(@PathParam("id") @NotNull final long id
       @QueryParam("start") @NotNull String startTimeIso,
       @QueryParam("end") @NotNull String endTimeIso,
       @QueryParam("force") @DefaultValue("false") String isForceBackfill,
-      @QueryParam("speedup") @DefaultValue("false") final Boolean speedup) throws Exception {
+      @QueryParam("speedup") @DefaultValue("false") final Boolean speedup,
+      @QueryParam("removeAnomaliesInWindow") @DefaultValue("false") final Boolean isRemoveAnomaliesInWindow) throws Exception {
```

Comment:
Put default to false since we don't want to mistakenly remove user labeled information. Here the wrapper endpoint is actually a use case when FIRST time do replay, which is assuming there is no anomalies exists in this anomaly function.

'force' is whether to resume backfill is there is already any.